Index: slurm-slurm-20-11-8-1/src/slurmrestd/plugins/openapi/dbv0.0.36/jobs.c
===================================================================
--- slurm-slurm-20-11-8-1.orig/src/slurmrestd/plugins/openapi/dbv0.0.36/jobs.c
+++ slurm-slurm-20-11-8-1/src/slurmrestd/plugins/openapi/dbv0.0.36/jobs.c
@@ -385,8 +385,7 @@ extern int op_handler_jobs(const char *c
 			 * default to grabbing all information
 			 * based on _init_params()
 			 */
-			.flags = (JOBCOND_FLAG_DUP | JOBCOND_FLAG_NO_TRUNC |
-				  JOBCOND_FLAG_WHOLE_HETJOB),
+			.flags = (JOBCOND_FLAG_DUP | JOBCOND_FLAG_NO_TRUNC),
 			.db_flags = SLURMDB_JOB_FLAG_NOTSET,
 		};
 		foreach_query_search_t args = {
@@ -413,8 +412,7 @@ static int _op_handler_job(const char *c
 	data_t *errors = populate_response_format(resp);
 	char *jobid;
 	slurmdb_job_cond_t job_cond = {
-		.flags = (JOBCOND_FLAG_DUP | JOBCOND_FLAG_NO_TRUNC |
-			  JOBCOND_FLAG_WHOLE_HETJOB),
+		.flags = (JOBCOND_FLAG_DUP | JOBCOND_FLAG_NO_TRUNC),
 		.db_flags = SLURMDB_JOB_FLAG_NOTSET,
 	};
 
