Index: slurm-slurm-20-11-8-1/src/slurmrestd/plugins/openapi/dbv0.0.36/parse.c
===================================================================
--- slurm-slurm-20-11-8-1.orig/src/slurmrestd/plugins/openapi/dbv0.0.36/parse.c
+++ slurm-slurm-20-11-8-1/src/slurmrestd/plugins/openapi/dbv0.0.36/parse.c
@@ -358,7 +358,7 @@ static const parser_t parse_job[] = {
 	_add_parse(UINT32, resvid, "reservation/id"),
 	_add_parse(UINT32, resv_name, "reservation/name"),
 	/* skipping show_full */
-	_add_parse(UINT32, eligible, "time/start"),
+	_add_parse(UINT32, start, "time/start"),
 	_add_parse(JOB_STATE, state, "state/current"),
 	_add_parse(JOB_REASON, state_reason_prev, "state/previous"),
 	_add_parse(UINT32, submit, "time/submission"),
